learn.tt

```diff
@@ -0,0 +1,146 @@
+          <!-- TODO
+          <a href="[%root%]/nixpkgs/manual/#">Visual Studio Code</a>,
+          -->
+          ...
+      </li>
+      <li><a href="[%root%]/nixpkgs/manual/#chap-submitting-changes">Contributing to Nixpkgs</a></li>
+    </ul>
+    <a href="[%root%]/nixpkgs/manual" class="btn btn-block">More ...</a>
+  </div>
+
+  <div class="span4">
+    <h2>Learn NixOS</h2>
+    <p>How to install, use and extend NixOS, a Linux distribution based on Nix package manager.</p>
+    <ul>
+      <li><a href="[%root%]/nixos/manual/#sec-installation">Installing NixOS</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-changing-config">Changing the Configuration</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-upgrading">Upgrading NixOS</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-configuration-syntax">Configuration Syntax</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-package-management">Package Management</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-user-management">User Management</a></li>
+      <li><a href="[%root%]/nixos/manual/#ch-troubleshooting">Troubleshooting</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-writing-modules">Writing NixOS Modules</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-writing-documentation">Writing NixOS Documentation</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-nixos-tests">Writing NixOS Tests</a></li>
+      <li><a href="[%root%]/nixos/manual/#sec-building-cd">Building Your Own NixOS CD</a></li>
+    </ul>
+    <a href="[%root%]/nixos/manual" class="btn btn-block">More ...</a>
+  </div>
+
+</div>
+
+<div class="row-fluid">
+
+  <div class="span6">
+    <h2>Other learning resources</h2>
+    <ul>
+      <li>
+        <a href="[%root%]/nixos/nix-pills">Nix Pills</a> —  At the beginning you may feel that some of the magic which happens behind the scenes is hard to grasp. This series aims to complement the existing explanations from the more This wiki is user-maintained. formal documents.</li>
+      <li><a href="https://nixos.wiki">Wiki</a> — A user-maintained wiki for Nix and NixOS.</li>
+      <li><a href="https://nix-cookbook.readthedocs.io">Nix cookbook</a> — Nix Community Cookbook presents topical, practical ways of using Nix package manager ecosystem.</li>
+    </ul>
+  </div>
+
+  <div class="span6">
+    <h2>Get help from the community</h2>
+    <ul>
+      <li><a href="https://discourse.nixos.org/">Discourse</a> — Discourse is a forum where we have discussions and help users out with their questions.</li>
+      <li><a href="irc://irc.freenode.net/#nixos">IRC</a> — You can drop by on the IRC channel <a href="irc://irc.freenode.net/#nixos"><strong><tt>#nixos</tt></strong></a> on <a href="https://freenode.net/"><tt>irc.freenode.net</tt></a> to chat with the developers or other users.</li>
+      <li><a href="https://stackoverflow.com/questions/tagged/nixos">Stackoverflow</a> — a place to ask questions.</li>
```

Comment:
when stackoverflow is listed we might also want to list reddit?

https://www.reddit.com/r/NixOS/

Reply:
:+1: